airflow/config_templates/default_airflow.cfg

```diff
@@ -16,1507 +15,21 @@
-
-# Used to set the maximum page limit for API requests. If limit passed as param
-# is greater than maximum page limit, it will be ignored and maximum page limit value
-# will be set as the limit
-maximum_page_limit = 100
-
-# Used to set the default page limit when limit param is zero or not provided in API
-# requests. Otherwise if positive integer is passed in the API requests as limit, the
-# smallest number of user given limit or maximum page limit is taken as limit.
-fallback_page_limit = 100
-
-# The intended audience for JWT token credentials used for authorization. This value must match on the client and server sides. If empty, audience will not be tested.
-# Example: google_oauth2_audience = project-id-random-value.apps.googleusercontent.com
-google_oauth2_audience =
-
-# Path to Google Cloud Service Account key file (JSON). If omitted, authorization based on
-# `the Application Default Credentials
-# <https://cloud.google.com/docs/authentication/production#finding_credentials_automatically>`__ will
-# be used.
-# Example: google_key_path = /files/service-account-json
-google_key_path =
-
-# Used in response to a preflight request to indicate which HTTP
-# headers can be used when making the actual request. This header is
-# the server side response to the browser's
-# Access-Control-Request-Headers header.
-access_control_allow_headers =
-
-# Specifies the method or methods allowed when accessing the resource.
-access_control_allow_methods =
-
-# Indicates whether the response can be shared with requesting code from the given origins.
-# Separate URLs with space.
-access_control_allow_origins =
-
-# Indicates whether the *xcomEntries* endpoint supports the *deserialize*
-# flag. If set to False, setting this flag in a request would result in a
-# 400 Bad Request error.
-enable_xcom_deserialize_support = False
-
-[lineage]
-# what lineage backend to use
-backend =
-
-[openlineage]
-
-# This section applies settings for OpenLineage integration.
-# For backwards compatibility with `openlineage-python` one can still use
-# `openlineage.yml` file or `OPENLINEAGE_` environment variables. However, below
-# configuration takes precedence over those.
-# More in documentation - https://openlineage.io/docs/client/python#configuration.
-# Set this to true if you don't want OpenLineage to emit events.
-disabled = False
-
-# OpenLineage namespace
-# Example: namespace = food_delivery
-# namespace =
-
-# Comma-separated paths to custom OpenLineage extractors.
-# Example: extractors = full.path.to.ExtractorClass;full.path.to.AnotherExtractorClass
-extractors =
-
-# Path to YAML config. This provides backwards compatibility to pass config as
-# `openlineage.yml` file.
-config_path =
-
-# OpenLineage Client transport configuration. It should contain type
-# and additional options per each type.
+#         airflow config list --defaults
 #
-# Currently supported types are:
+# You can redirect it to your configuration file and edit it:
```

Comment:
Good point. At least start edifucating people here is a good idea.

Reply:
I actually did even more than that. The "--defaults" option now containts all the variables commented out by .... default :) . This is really nice configuration that is "starting point" - because you can still see all the generated descriptions, examples, variables, but the actual values are all commented out, which means that you can selectively uncomment them (and change). I'v also copied this description to the user docs "howto/set-config.rst" and explained this is the approach recommended for production.

Reply:
Perfect, thanks! I missed that detail...

Reply:
It was not like that before. I added it after your comment when I realized we can finally have a tool to help users do what we always said is a good idea,  so credits go to you for putting me on the path :)